### PR TITLE
Latk format export

### PR DESCRIPTION
**Pull request by n1ckfg**
Lightning Artist Toolkit files (for use with Blender Grease Pencil, and volumetric animation interchange generally) added as an export format.

### CODE REVIEW

File: Assets/Scripts/SketchMemoryScript.cs [hunk 1]
```diff
@@ -1155,4 +1170,4 @@ private static void UpdateTimestampsToCurrentSketchTime(Stroke stroke)
             }
         }
     }
-} // namespace TiltBrush
+} // namespace TiltBrush
```
Comment (mikeskydev) [inline Assets/Scripts/SketchMemoryScript.cs:1173]: nit: line ending

File: ProjectSettings/ProjectVersion.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.25f1
```
Comment (mikeskydev) [inline ProjectSettings/ProjectVersion.txt:1]: It probably won't effect anything but can/should this happen in a separate PR? the CI environment variables would also need updating to match this if we do it here.

File: Assets/Scripts/Export/ExportLatk.cs [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+                outputMemStream.Read(bytes, 0, (int)outputMemStream.Length);
+                file.Write(bytes, 0, bytes.Length);
+                outputMemStream.Close();
+            }
+        }
+
+    }
+
+}
+#endif
```
Comment (mikeskydev) [inline Assets/Scripts/Export/ExportLatk.cs:247]: nit: line ending

File: Assets/PlatformConfigPC.asset [hunk 4]
```diff
@@ -28,7 +28,8 @@ MonoBehaviour:
   EnableExportFbx: 1
   EnableExportGlb: 1
   EnableExportJson: 1
-  EnableExportUsd: 1
```
Comment (mikeskydev) [inline Assets/PlatformConfigPC.asset:31]: Why was USD disabled?

**Review by mikeskydev — COMMENTED** [review]
Just a few nitpicks! Also: should latk export be enabled for Quest builds or do you expect this to be mainly used on desktop builds?

File: Assets/PlatformConfigPC.asset (hunk 4 above)
Comment (n1ckfg) [reply Assets/PlatformConfigPC.asset:31]: Probably superstition; I got a strange error from the build system about USD on the next-to-last commit. Put it back now.

**Comment by n1ckfg** [comment]
That should be all the notes! Re: Quest, the exporter code and dependencies should be OK for any Android headset; I've used them on various projects before for ML1, Mirage, and Quest 1.

**Comment by andybak** [comment]
I think at some point we'll need to provide a UI for users to choose the export formats but at the moment it's not so bad that we can't add another one.

(maybe we should enable a few common ones by default and allow the others via a config file override)

**Comment by n1ckfg** [comment]
A user-editable config file is a great idea

**Pull request merged**